### ISSUE TITLE
Is 0.2.5 yarn only?

### ISSUE THREAD

**Issue by photonstorm** [mitschabaude/esbuild-plugin-wat#1]
Have been using v0.2.3 via npm quite happily. Noticed an upgrade to 0.2.5 but it gives the following error using `npm i`:

```
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "link:": link:.yalc/@webassemblyjs/ast
```

I'm using npm 7.21.1

**Comment by mitschabaude** [comment]
Thanks for reporting! I thought I had it working with both npm and yarn but I seem to have missed something (I'm on similar versions of npm).

Yeah, so I think it was working with `file:...` in npm but not in yarn, then I discovered it was working with `link:...` in yarn, changed it and didn't check again properly for npm. My bad.

For context, I have a pretty weird package layout currently but hopefully that's only temporary. The reason is that I'm working on some advanced new bundling / treeshaking functionality for Wasm and, to make it work, I had to fix some issues in a library I rely on, [xtuc/webassemblyjs](https://github.com/xtuc/webassemblyjs). Because these fixes aren't yet merged into their npm packages, I used [yalc](https://github.com/wclr/yalc) to inline my local, patched webassemblyjs packages in this repo directly and declared them as bundledDependencies. It magically seemed to work (but doesn't apparently).

I think the more straight-forward way (but less flexible) would be to install WIP packages from a github branch. However, this doesn't seem to be supported when the package is in a subfolder of the github repo, which is the case here.

So... if anyone knows a simple solution for this, I'd be really interested! I'll try to figure it out over the next couple of weeks.

As a quick solution, I recommend to stick with 0.2.3 for now :)
